presentation/src/main/java/com/sopt/presentation/auth/login/LoginViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package com.sopt.presentation.auth.login
+
+import com.sopt.core.util.BaseViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class LoginViewModel @Inject constructor() : BaseViewModel<LoginSideEffect>() {
+
+    fun navigateToSignup() {
+        emitSideEffect(LoginSideEffect.NavigateSignUp("authId"))
+    }
```

Comment:
```suggestion
    fun navigateToSignup(authId: String) {
        emitSideEffect(LoginSideEffect.NavigateSignUp(authId))
    }
```
P1: 매개변수를 줘야 나중에 값 전달할 때 편할 것 같습니다